.github/workflows/dependency_check.yaml

```diff
@@ -0,0 +1,49 @@
+name: Check For Unused Dependencies
+
+on:
+  schedule:
+    - cron: '0 8 1,15 * *' # At 8am on 1st and 15th of each month
```

Comment:
Do we know what timezone 8am is? I don't think it matters but I just thought I'd ask.
I did double check the cron string and it looks correct.

Reply:
[These docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) state that it's UTC, so right now this will run @ 4 AM twice a month